pkgs/top-level/perl-packages.nix

```
@@ -14747,6 +14762,52 @@ let
     };
   };
 
+  PDL = buildPerlPackage rec {
+    pname = "PDL";
+    version = "2.019";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/C/CH/CHM/${pname}-${version}.tar.gz";
+      sha256 = "51a47beedb1dd5bb4b6f6441a0ad38d4c95d7cbf3bf6b1b4c9128732ea2d77fe";
```

Latest PDL is maintained by EDJ https://metacpan.org/pod/PDL

```suggestion
      url = "mirror://cpan/authors/id/E/ET/ETJ/${pname}-${version}.tar.gz";
      sha256 = "1szgyivnjiqrlclvh14ccg3vyfzlnz59bjqchlcrjbkirkd3y331";
```